Antidote/ActiveCallViewController.m

```diff
@@ -153,17 +159,53 @@ - (void)installConstraints
 
 }
 
-#pragma mark - Private
+#pragma mark - Public
 
-- (void)didUpdateCall
+- (void)setCallDuration:(NSTimeInterval)callDuration
 {
-    [super didUpdateCall];
+    _callDuration = callDuration;
     [self updateTimerLabel];
 }
 
+- (void)setMicSelected:(BOOL)micSelected
+{
+    self.microphoneButton.selected = micSelected;
+}
+
+- (BOOL)micSelected
+{
+    return self.microphoneButton.selected;
+}
+
+- (void)setSpeakerSelected:(BOOL)speakerSelected
+{
+    self.speakerButton.selected = speakerSelected;
+}
+
+- (BOOL)speakerSelected
+{
+    return self.speakerButton.selected;
+}
+
+- (void)setShowIncomingCallView:(BOOL)showIncomingCallView
+{
+    if (showIncomingCallView) {
+        [self setupIncomingCallView];
```

Comment:
CallsManager should already be avoiding that situation. However I am not sure if I should also find a way to protect that from happening here?